Branch-SDK/src/main/java/io/branch/referral/PrefHelper.java

```diff
@@ -54,7 +54,9 @@
      */
     private static final int MAX_RETRIES = 3; // Default retry count is 3
 
-    static final int TIMEOUT = 5500; // Default timeout id 5.5 sec
+    static final int TIMEOUT = 5500; // Default timeout is 5.5 sec
+    static final int CONNECT_TIMEOUT = 10000; // Default timeout is 10 seconds
```

Comment:
10sec seems excessive as a default? However, I have no basis for this statement. 🤣

Reply:
Discussed internally, these values are approx equal to the ios implementation